app/scripts/modules/core/src/presentation/LinkWithClipboard.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CopyToClipboard } from 'core/utils';
+
+export interface ILinkWithClipboardProps {
+  url: string;
+  text: string;
+}
+
+export const LinkWithClipboard = ({ url, text }: ILinkWithClipboardProps) => (
+  <>
+    <a href={url} target="_blank">
```

Comment:
Would `target="_blank"` be a reasonable expectation for users or should we make it configurable?

Reply:
I think this is a reasonable expectation for users. This component will be used mainly for external links, in which case we would want a new page.